.github/workflows/connect-webextension.test.yml

```diff
@@ -22,6 +22,8 @@ jobs:
       - run: yarn install
       - run: yarn build:libs
       - name: Build extension
-        run: "node ./packages/connect-examples/update-webextensions.js"
+      # this does not make much sense, in fact, we are testing what is alredy in production and we cant change that
+      # unless we have build and deploys in github
+        run: "node ./packages/connect-examples/update-webextensions.j --npm-src="https://connect.trezor.io/9/trezor-connect.js"
```

Comment:
It looks like there are some typos here. 
* `update-webextensions.j` should be `update-webextensions.js`
* it shoudl end with other " to close the initial one
It should be like:

```
"node ./packages/connect-examples/update-webextensions.js --npm-src="https://connect.trezor.io/9/trezor-connect.js""
```